src/backend/expungeservice/charge_classifier.py

```diff
@@ -236,6 +229,8 @@ def _civil_offense(statute, name):
             return AmbiguousChargeTypeWithQuestion([CivilOffense])
         elif "fugitive" in name:
             return AmbiguousChargeTypeWithQuestion([CivilOffense])
+        elif "contempt" in name:
```

Comment:
Have you double checked this is a safe abbreviation compared to "contempt of court"?

Reply:
Yea, but also every contempt of court case I found says "contempt of court", so this may as well be more specific